Code/ObjectMapping/RKObjectMapping.m

```diff
@@ -69,8 +69,8 @@ - (RKObjectMapping *)invertMapping:(RKObjectMapping *)mapping withPredicate:(BOO
     inverseMapping = [RKObjectMapping requestMapping];
     (self.invertedMappings)[dictionaryKey] = inverseMapping;
     [inverseMapping copyPropertiesFromMapping:mapping];
-    // We want to serialize `nil` values
-    inverseMapping.assignsDefaultValueForMissingAttributes = YES;
+    // We potentially want to serialize `nil` values, adopt the strategy from the mapping we're inverting.
```

Comment:
That's intentional, please revert this part of your PR and I can merge. https://github.com/RestKit/RestKit/commit/1177f8075cce0bf1d62f849b8408ad2ebe23bb17